newrelic-agent/src/main/java/com/newrelic/agent/config/IBMUtils.java

```diff
@@ -11,17 +11,21 @@
 import java.util.regex.Pattern;
 
 public class IBMUtils {
+    private static final String IBM_VENDOR = "IBM Corporation";
```

Comment:
Does this also catch the case of an Eclipse OpenJ9 VM?

Reply:
@kittylyst  I have no context around this and I'm happy to dig into it in the future. Since we are committed to finish by EOD, I don't know if I'll be able to fully grok what this means by the deadline. 

Could you make an issue to follow up on this and share whatever context you may have? 

@tspring  In case you have context, maybe you could provide a better answer here?

Reply:
Done - https://github.com/newrelic/newrelic-java-agent/issues/227